app/controllers/geocoder_controller.rb

```diff
@@ -101,9 +101,13 @@ def search_osm_nominatim
       if klass == "boundary" && type == "administrative"
         rank = (place.attributes["address_rank"].to_i + 1) / 2
         prefix_name = t "geocoder.search_osm_nominatim.admin_levels.level#{rank}", :default => prefix_name
+        border_type = false
+        place_type = false
```

Comment:
I think `nil` would be a better initial value here given they are going to be strings if any match is found.